schema/crdb/visibility/schema.sql

```diff
@@ -0,0 +1,121 @@
+CREATE EXTENSION IF NOT EXISTS btree_gin;
+
+-- convert_ts converts a timestamp in RFC3339 to UTC timestamp without time zone.
+CREATE FUNCTION convert_ts(s VARCHAR) RETURNS TIMESTAMP AS $$
```

Comment:
@Hades32 need to remove this

Reply:
ah yeah, these files aren't really executed that's why I didn't notice. Will clean it up